k9mail/src/main/java/security/planck/ui/passphrase/manage/compose/PassphraseManagementScreen.kt

```diff
@@ -289,15 +291,16 @@ private fun ManageScreenButtonsRow(
         )
 
         TextActionButton(
-            text = stringResource(id = R.string.previous_action),
+            text = stringResource(id = R.string.manage_identities_remove_action),
```

Comment:
when did we rename these strings? It's fine of course, they just don't seem to be into this review

Reply:
I was just re-using a string resource we already have (identities_remove_action), not the best practice but it is already localized. I would say better to generalize the name as "action_remove" or so.